datafusion/core/src/datasource/physical_plan/mod.rs

```diff
@@ -35,7 +35,7 @@ pub use self::parquet::{ParquetExec, ParquetFileMetrics, ParquetFileReaderFactor
 
 pub use arrow_file::ArrowExec;
 pub use avro::AvroExec;
-pub use csv::{CsvConfig, CsvExec, CsvOpener};
+pub use csv::{CsvConfig, CsvExec, CsvExecBuilder, CsvOpener};
```

Comment:
this struct, added by @connec  recently, actually wasn't publically exported which was flagged by one of the doc tests